lib/alertify.js

```
@@ -199,16 +199,29 @@
 
 		/**
 		 * Close the log messages
-		 * 
+		 *
+		 * @param  {Object} elem    HTML Element of log message to close
+		 * @param  {Number} wait    [optional] Time (in ms) to wait before automatically hiding the message  
+		 *
 		 * @return {undefined}
 		 */
-		close = function (elem) {
+		close = function (elem, wait) {
+			var timer;
+			if (wait && typeof wait !== undefined){
+				if (!isNaN(wait)){
+					timer = +wait;
+				} else {
+					throw new Error("Log message delay must be a Number");
+				}
+			} else {
+				timer = delay;
+			}
```

I was thinking a 1 line addition, don't like how it adds 10 lines. Maybe a silent failure is better (as you originally had). I do like the testing of the wait number though. I would comment the unary plus since it's not obvious what it does (for everyone)

``` js
// +wait -> Unary Plus,  evaluates operand and attempts to converts it into a number
var timer = (wait && !isNaN(wait)) ? +wait : delay
```

my bad